Dockerfile

```diff
@@ -9,11 +9,13 @@ RUN npm install
 
 COPY . .
 
+RUN npm run tsc
```

Comment:
I think neither development or production needs this? 🤔 If I understood correctly in dev you don't need this because you use `ts-node` and in production this thing is already been done by `compile` step in circleCI